src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksNewVendorRequestsServiceImpl.java

```diff
@@ -121,23 +121,36 @@ private boolean pmwNewVendorRequestProcessingIntoKfsWasSuccessful(PaymentWorksVe
         }
     }
     
-    private boolean canPaymentWorksNewVendorRequestProcessingContinueForVendor(PaymentWorksVendor stgNewVendorRequestDetailToProcess, PaymentWorksNewVendorRequestsBatchReportData reportData) {
+    private boolean canPaymentWorksNewVendorRequestProcessingContinueForVendor(PaymentWorksVendor stgNewVendorRequestDetailToProcess, 
+            PaymentWorksNewVendorRequestsBatchReportData reportData) {
+
         List<String> errorMessages = new ArrayList<String>();
         if (pmwDtosCouldConvertCustomAttributesToPmwJavaClassAttributes(stgNewVendorRequestDetailToProcess) &&
-            pmwNewVendorAttributesConformToKfsLengthsOrFormats(stgNewVendorRequestDetailToProcess, errorMessages) &&
-            allPmwNewVendorIsoCountriesMapToSingleFipsCountry(stgNewVendorRequestDetailToProcess, errorMessages) &&
-            pmwNewVendorIdentifierDoesNotExistInKfsStagingTable(stgNewVendorRequestDetailToProcess, errorMessages)){
+                pmwNewVendorAttributesConformToKfsLengthsOrFormats(stgNewVendorRequestDetailToProcess, errorMessages) &&
+                validateVendorType(stgNewVendorRequestDetailToProcess, errorMessages) && 
```

Comment:
I have taken another look at this. Upon further consideration, it may be more appropriate to include this check in method pmwNewVendorAttributesConformToKfsLengthsOrFormats where the rest of these types of items are validated instead of having an additional boolean check in the if-clause.